pkg/broker/config/targets.proto

```diff
@@ -87,6 +87,9 @@ message Target {
   // The resolved subscriber URI of the target.
   string address = 5;
 
+  // The resolved URI that replies are sent to.
+  string reply_address = 10;
```

Comment:
When you say `reorder` do you mean altering the numbers, such that reply_address is 6 rather than 10? Or moving the fields in this file to match the number order? Or something else?

Reply:
I was thinking about moving fields to match the number order, so that we don't have to find the max number manually next time we add one.

Reply:
Done.